src/lib/components/Subscribe/index.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * @fileoverview Element that renders newsletter subscription form.
+ */
+
+import {BaseElement} from "../BaseElement";
+import "./_styles.scss";
+
+/**
+ * Element that renders newsletter subscription form.
+ *
+ * @extends {BaseElement}
+ * @final
+ */
+class Subscribe extends BaseElement {
+  constructor() {
+    super();
+
+    this.form = this.querySelector("form");
```

Comment:
Suggest using `.w-subscribe__form` here. I figure we won't have more than one form inside of this element but may as well play it safe.